configuration/config-txt/overclocking.md

```diff
@@ -53,12 +53,10 @@ This table gives the default values for the options on various Raspberry Pi Mode
 | v3d_freq_min |  250   | 250 | 250 | 250   | 250   | 500  |
 | sdram_freq_min |400   | 400 | 400 | 400   | 400   | 400  |
 
-| gpu_freq | Sets `core_freq`, `h264_freq`, `isp_freq`, and `v3d_freq` together; on Pi 3/Pi Zero /Pi Zero W `core_freq` defaults to `400`, while `h264_freq`, `isp_freq`, and `v3d_freq`default to `300`; on Pi 4B this can be set to 500 or 600 only|
```

Comment:
Now that this text has been removed, it seems there's now nowhere on the page that says that 500 and 600 are the only valid settings for gpu_freq / core_freq on the Pi4B?
However I've never overclocked any of my Pis, so I dunno which overclocking information on this page is important or not :man_shrugging:

Reply:
linking in another convo here https://github.com/raspberrypi/firmware/issues/1308

I question if 500 and 600 being the only valid settings is a true statement.

e.g. I can do this on my RPi4
```
force_turbo=1
gpu_freq=250
gpu_freq_min=250
```
and confirm that `vcgencmd measure_clock core` shows 250MHz under load/idle conditions

Reply:
I'm just waiting for input from the guy who knows all about this before finishing off this PR. The possible options changed when we moved to the DVFS firmware and its subsequent updates, so I think the 500/600 stuff is incorrect.

Reply:
@popcornmix Assuming core_freq is fixed on the PI4, what options do we have for the remaining options - `h264_freq`, `isp_freq`, and `v3d_freq`?

Reply:
Also hevc_freq. Basically you can ask for what you want. The higher of the 4 will determine the channel frequency. The PLLA frequency will use the max integer multiple that doesn't exceed 3GHz. The other 3 clocks just get an integer divider from channel frequency.

So you really want to set them to all be the same (e.g. through gpu_freq) unless you are happy with some being at half that rate.

Reply:
Is `hevc_freq` available on all Pis, or is it a Pi4-only setting?

Reply:
So presumably gpu_freq no longer sets core_freq - is that change only on Pi4? And is the implications of what you say that you can set these numbers up to 3GHz? Obviously things break before then, but is that the theoretical max? Quick google says people can usually get to 750 before things stop working.

Reply:
hevc_freq only affects pi4 (no hevc hardware on earlier models)

gpu_freq affects all gpu clocks including  core, but due to limitations on channels and dividers, it will almost always be a no-op.

As requesting core_freq=750 has no effect, then requesting gpu_freq=750 will not affect core frequency.

Strictly gpu_freq=750/core_freq=500 would be better but they both will behave the same with current firmware.

I didn't say you can set clocks to 3GHz - the PLL can go up to 3GHz but there are two dividers between PLL, channel freq and clock.

As I keep saying this is just background information for the ultra curious. It shouldn't go in the top level documentation, because it is an implementation detail that is subject to change and of no use to anyone.

Overclocking core frequency is unsupported on Pi4. You can overclock v3d/h264/hevc/isp.